docs/platform/tizen-studio/overview.md

```diff
@@ -35,8 +35,8 @@ If you have found a bug, have any questions, or want to contribute, feel free to
 
 - [Tizen Studio Download](https://developer.tizen.org/development/tizen-studio/download)
 - [OpenSource Repository](https://review.tizen.org/gerrit/#/)
-- [Tool Guide](https://developer.tizen.org/development/tizen-studio/native-tools)
-- [Training](https://developer.tizen.org/development/training/overview)
-- [API References](https://developer.tizen.org/development/api-references/native-application)
-- [Release Notes](https://developer.tizen.org/development/tizen-studio/download/release-notes)
+- [Tool Guide](../../application/tizen-studio/native-tools/index.md)
+- [Get Started for Native Application](../../application/native/get-started/overview.md)
+- [Get Started for Web Application](../../application/native/get-started/overview.md)
```

Comment:
This link to Web is actually going to native.

Reply:
Done.